network-area-diagram/src/main/java/com/powsybl/nad/build/iidm/NetworkGraphBuilder.java

```diff
@@ -104,6 +105,12 @@ private void visitThreeWindingsTransformer(VoltageLevel vl, ThreeWindingsTransfo
         }
     }
 
+    private void visitDanglingLine(DanglingLine dl, Graph graph) {
+        BoundaryNode boundaryNode = new BoundaryNode(idProvider.createId(dl), dl.getId(), dl.getNameOrId());
+        graph.addNode(boundaryNode);
+        addDanglingLineEdge(graph, dl, boundaryNode);
```

Comment:
I think it is useful to have them. They are important nodes, as they connect the network to its "boundaries". In IIDM are modelled as "dangling" because we do not want to model explicitly the boundary bus, but they are regular AC line segments.